applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/volume/bt_vol_ctlr.c

```diff
@@ -10,14 +10,22 @@
 #include <zephyr/types.h>
 #include <zephyr/bluetooth/conn.h>
 #include <zephyr/bluetooth/audio/vcp.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <../../../../subsys/bluetooth/audio/vcp_internal.h>
```

Comment:
Suggest to add a comment why you are including the internal header file, and possibly add a reference to https://github.com/zephyrproject-rtos/zephyr/issues/71614